traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -23,6 +23,9 @@ import (
 	"database/sql"
 	"errors"
 	"fmt"
+	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/topology/topology_validation"
```

Comment:
nit: these imports are grouped haphazardly

Reply:
Organized imports in 564d00922f